model-optimizer/extensions/front/MatMul_normalizer.py

```diff
@@ -90,6 +91,10 @@ def replace_sub_graph(self, graph: Graph, match: [dict, SubgraphMatch]):
         node.out_port(0).get_connection().set_source(bias_node.out_port(0))
         node.in_port(2).get_connection().set_destination(bias_node.in_port(1))
         node.out_port(0).connect(bias_node.in_port(0))
+        if not bias_node.in_port(1).get_source():
+            bias_const_node = Const(graph, {'name': name + '/Const',
+                                            'value': 0}).create_node()
+            bias_const_node.out_port(0).connect(bias_node.in_port(1))
```

Comment:
I don't think that we need it at all if `bias` is zero. If bias is zero, don't insert `Add` node at all.
Here also we have a problem with dtype, we don't know dtype of this Constant node.